ui/src/layout/Menu.js

```diff
@@ -84,7 +91,9 @@ const Menu = ({ onMenuClick, dense, logout }) => {
     resource.hasList && resource.options && resource.options.subMenu === subMenu
 
   return (
-    <div>
+    <div
+      className={open ? classes.fixedSidebarOpen : classes.fixedSidebarClose}
```

Comment:
We can pass `open` as a param to `useStyles` and check this condition inside it. Makes the code cleaner. Example: https://github.com/navidrome/navidrome/blob/ac37bf3631610747001f44fe843517f3fdf0ac08/ui/src/common/LoveButton.js#L12-L13

Reply:
Yes, we can do that and I'll make the necessary changes to it.